bin/package.xml.in

```diff
@@ -55,5 +55,11 @@ necessary to build a fully-functional MongoDB driver.
   </required>
  </dependencies>
  <providesextension>mongodb</providesextension>
- <extsrcrelease/>
+ <extsrcrelease>
+  <configureoption name="with-libbson" default="no" prompt="Use system libbson"/>
+  <configureoption name="with-libmongoc" default="no" prompt="Use system libmongoc"/>
+  <configureoption name="with-openssl-dir" default="yes" prompt="OpenSSL dir for MongoDB"/>
+  <configureoption name="with-system-ciphers" default="no" prompt="Use system default cipher list instead of hardcoded value"/>
```

Comment:
This may also warrant a note that it's for OpenSSL. Perhaps "OpenSSL: Use system default..."?